From 0cbfc2cfec0b8fbd15dfc7d863c0396b22b34217 Mon Sep 17 00:00:00 2001
From: Mathieu Bivert <mathieu.bivert@intel.com>
Date: Tue, 23 Apr 2013 08:41:45 +0200
Subject: [PORT FROM MAIN]wifi/crashtool: wpa supplicant log association to AP (bcm)

BZ: 102710

Log access point related informations when connecting to a wifi network.

Change-Id: I9575ac2ea97406f5832ff1759882637e3c374351
Orig-Change-Id: If0c8bc3bdd30edd21dc7f29a961bd56068c690d7
Category: device enablement
Domain: CWS.WIFI-Common
Origin: upstream
Upstream-Candidate: no,proprietary
Signed-off-by: Mathieu Bivert <mathieu.bivert@intel.com>
Reviewed-on: http://android.intel.com:8080/122744
Reviewed-by: buildbot <buildbot@intel.com>
Reviewed-by: Fiat, Christophe <christophe.fiat@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 wpa_supplicant/wpa_supplicant.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/wpa_supplicant/wpa_supplicant.c b/wpa_supplicant/wpa_supplicant.c
index 5117a18..eea4e8c 100644
--- a/wpa_supplicant/wpa_supplicant.c
+++ b/wpa_supplicant/wpa_supplicant.c
@@ -620,7 +620,7 @@ static unsigned long timevaldiff(struct timeval *starttime, struct timeval *fini
 	return msec;
 }
 
-static inline void log_connection_time(struct wpa_supplicant *wpa_s,
+static inline void log_connection(struct wpa_supplicant *wpa_s,
                                       enum wpa_states state)
 {
 	struct ct_event *ev = NULL;
@@ -656,7 +656,7 @@ static inline void log_connection_time(struct wpa_supplicant *wpa_s,
 		snprintf(bssid, sizeof(bssid), "%s (" MACSTR ") / %d MHz",
 			wpa_s->current_ssid->ssid, MAC2STR(wpa_s->bssid), wpa_s->assoc_freq);
 
-		lct_log(CT_EV_STAT, "cws.wifi", "wpa_connected", EV_FLAGS_PRIORITY_LOW,
+		lct_log(CT_EV_STAT, "cws.wifi", "connected", EV_FLAGS_PRIORITY_LOW,
 			msec_c, bssid, sec);
 
 		start.tv_sec = 0;
@@ -705,7 +705,7 @@ void wpa_supplicant_set_state(struct wpa_supplicant *wpa_s,
 		wpa_supplicant_state_txt(wpa_s->wpa_state),
 		wpa_supplicant_state_txt(state));
 
-	log_connection_time(wpa_s, state);
+	log_connection(wpa_s, state);
 
 #ifdef ANDROID_P2P
 	if(state == WPA_ASSOCIATED && wpa_s->current_ssid) {
-- 
1.7.4.1

